edx-platform/cabinet-theme/lms/templates/static_templates/server-error.html

```diff
@@ -0,0 +1,36 @@
+<%page expression_filter="h"/>
```

Comment:
This theme folder is not part of the Pearson available themes, please remove change from this theme. @luismorenolopera

Reply:
done